akka-remote/src/main/scala/akka/remote/serialization/ArteryControlMessageSerializer.scala

```diff
@@ -144,17 +146,25 @@ private[akka] class ArteryControlMessageSerializer(val system: ExtendedActorSyst
   def serializeSystemMessageEnvelope(env: SystemMessageDelivery.SystemMessageEnvelope): ArteryControlFormats.SystemMessageEnvelope = {
     val msg = MessageSerializer.serialize(system, env.message)
```

Comment:
This will actually create bytes with `SerializedMessage`, which contains the serializerId and manifest.
Here is how we have done it in other places:

https://github.com/akka/akka/blob/master/akka-remote/src/main/scala/akka/remote/serialization/MiscMessageSerializer.scala#L55

https://github.com/akka/akka/blob/master/akka-remote/src/main/scala/akka/remote/serialization/MiscMessageSerializer.scala#L132